auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -40,8 +40,8 @@
             </td>
             <td v-if="showCol(headers[3].text)">{{ status(item) }}</td>
             <td v-if="showCol(headers[4].text)">{{ folio(item) }}</td>
-            <td v-if="showCol(headers[5].text)">{{ lastModified(item) }}</td>
-            <td v-if="showCol(headers[6].text)">{{ modifiedBy(item) }}</td>
+            <!-- <td v-if="showCol(headers[5].text)">{{ lastModified(item) }}</td> -->
+            <!-- <td v-if="showCol(headers[6].text)">{{ modifiedBy(item) }}</td> -->
```

Comment:
Hey reviewers, do you think we should delete this instead?

Reply:
![image](https://user-images.githubusercontent.com/51458187/167220070-015aa4bc-a8bd-4951-a00c-cdbca5431cac.png)

Correct me if I'm wrong. I think this ticket is about hiding the two columns in default. User has the option to display upon checking the boxes later.

Reply:
Good catch. Yes, I'm unsure whether we should comment these out or just hide them.

@lambert-alex Could you please with Linda on this? In retrospect, it does seem that hiding these by default is simpler than deleting them.

Reply:
I thought we wanted to remove the user from even being able to see it as apparently it displays wrong information for PPR users. I did send Linda a message about it.